services/online_exam_service/.env

```
@@ -1,18 +1,29 @@
-# DATABASE CONFIG (k?t n?i d?n course_db)
+# PostgreSQL Database (shared with exam-service)
 DB_HOST=localhost
 DB_PORT=5433
 DB_USER=postgres
 DB_PASSWORD=password
 DB_NAME=exam_db
 
-# BLOCKCHAIN CONFIG
+# Blockchain (Ganache)
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
+OWNER_ACCOUNT_PRIVATE_KEY=0x855503bbff30f22be58aa5a251d8beea293c56abe91b9ebf7733158790739a6c
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OWNER_ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)